source/MaterialXGenShader/OCIOColorManagementSystem.cpp

```diff
@@ -222,11 +398,18 @@ ImplementationPtr OCIOColorManagementSystem::getImplementation(const ColorSpaceT
         // a valid function name is created here
         std::string transformFunctionName = "IM_" + createValidName(transform.sourceSpace) + "_to_" + createValidName(transform.targetSpace) + "_" + typeName + "_ocio";
         shaderDesc->setFunctionName(transformFunctionName.c_str());
+        shaderDesc->setResourcePrefix(transformFunctionName.c_str());
         
         // Retrieve information
         gpu->extractGpuShaderInfo(shaderDesc);
 
         std::string fullFunction = shaderDesc->getShaderText();
+        // TODO: Comment out uniforms as a workaround as the uniforms are always embedded as
+        // part of the function string
+        StringMap uniformDecl;
+        uniformDecl["uniform"] = "//uniform";
+        fullFunction = replaceSubstrings(fullFunction, uniformDecl);
```

Comment:
This is purely a workaround for now to comment out uniforms as they are extracted out and inserted later one. This works only for GLSL which is currently the only supported language.